config/runtime.exs

```diff
@@ -2,13 +2,39 @@ import Config
 
 env = config_env()
 
+# Chain
+config :aecore, network_id: System.get_env("NETWORK_ID", "ae_mainnet")
```

Comment:
This is dev only without docker, keeping it in `dev.exs` would avoid conflicts in prod.

Reply:
Move this back to dev.config. Thanks